qlty-plugins/plugins/linters/rubocop/fixtures/plugins.in/.qlty/qlty.toml

```diff
@@ -1,5 +1,8 @@
 config_version = "0"
 
+[runtimes.enabled]
+ruby = "3.4.7"
```

Comment:
This basically serves as a test, and I did double check locally in the tool cache.